velox/docs/functions/spark/string.rst

```diff
@@ -69,12 +76,28 @@ Unless specified otherwise, all functions return NULL if at least one of the arg
     as being relative to the end of the string. Type of 'start' must be an INTEGER. 
 
 .. spark:function:: substring(string, start, length) -> varchar
+    :noindex:
 
     Returns a substring from ``string`` of length ``length`` from the starting
     position ``start``. Positions start with ``1``. A negative starting
     position is interpreted as being relative to the end of the string.
     Type of 'start' must be an INTEGER. ::
+
         SELECT substring('Spark SQL', 5, 1); -- k
         SELECT substring('Spark SQL', 5, 0); -- ""
         SELECT substring('Spark SQL', 5, -1); -- ""
-        SELECT substring('Spark SQL', 5, 10000); -- "k SQL"
\ No newline at end of file
+        SELECT substring('Spark SQL', 5, 10000); -- "k SQL"
+
+.. spark:function:: replace(string, search, replace) -> string
```

Comment:
Functions should appear in alphabetical order. Please, move.

Reply:
Just fixed it.